### DISCUSSION TITLE
Sidechain compressor of input & output

### DISCUSSION THREAD

**Discussion by AlexWayfer** [wwmm/easyeffects#973] [Q&A]
Hello.

I'm a newbie in sound effects and management.

I know that there is a possibility in [OBS](https://obsproject.com/) to create a sidechain compressor for lowering output volume on input signal.

I see "Compressor" with "Sidechain" mode in PulseEffects, for output and for input. But I don't see any "Source" option and my poke method with toggling everything didn't help.

So, I'm curious, can I somehow achieve [the same effect as in OBS](https://youtu.be/WK1PES5GTK4), but at system level, via PE?

**Comment by wwmm** [comment]
That particular sidechain compressor we are using does not allow a microphone to be set as source. Besides this one `Linux Studio Plugins` has another sidechain compressor that may allow this. And `Calf Studio` probably has one that can do this too. The problem is that GStreamer can not even load these "more advanced" sidechain compressors. So I never could try any of them.

Now that I am using native PipeWire plugins in the gtk4 branch I was forced to write my own wrapper for `LV2` plugins. Maybe now I will have the chance to use the other sidechain compressors.

**Comment by wwmm** [comment]
I suggest you open an issue about this. It will help me to remember to try these other compressors.

**Reply by AlexWayfer** [reply]
OK, thanks, created #974. Just wanted to be sure that this is suitable question/request.
